server/router.go

```diff
@@ -0,0 +1,46 @@
+package server
+
+import (
+	"net/http"
+
+	"github.com/ayushjnv1/Gobank/account"
+	"github.com/ayushjnv1/Gobank/api"
+	"github.com/ayushjnv1/Gobank/transaction"
+	"github.com/ayushjnv1/Gobank/user"
+	"github.com/gorilla/mux"
+)
+
+const (
```

Comment:
storre it in a common place so that any piece of code that needs these can access easily

and also name it likee "ProfileAdmin"
'ProfileUser" and maybe try to use iota here